src/tests/plugins/factories/stripe-integration-factory.test.js

```diff
@@ -7,6 +7,10 @@ const makeSut = () => {
   }
 }
 
+const fixtures = {
+  stripeToken: 'token_test_1234556'
+}
+
 describe('StripteIntegrationFactory', () => {
```

Comment:
add a scenario with the error, to ensure that we have a defined behavior when a token is not provided.